Brokerages/Tradier/TradierBrokerage.cs

```diff
@@ -1094,8 +950,6 @@ public override bool CancelOrder(Order order)
         public override void Connect()
         {
             _disconnect = false;
```

Comment:
Think `_disconnect` could be volatile, there are a couple of while loops in the `private IEnumerable<TradierStreamData> Stream(List<string> symbols)` maybe they should check this flag?